tests/sdk/java/testdata/await/generic/crds.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: genericawaiters.test.pulumi.com
+spec:
+  group: test.pulumi.com
+  names:
+    kind: GenericAwaiter
+    listKind: GenericAwaiterList
+    plural: genericawaiters
+    singular: genericawaiter
+  scope: Namespaced
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                someField:
+                  type: string
+            status:
+              type: object
+              properties:
```

Comment:
Consider adding an `observedGeneration` property to trigger additional logic within kstatus.